### PR TITLE
Added an appdata.xml file for Linux software gallery integration

### PR DESCRIPTION
**Pull request by Mailaender**
Rejected at https://github.com/aseprite/aseprite/pull/1233. You may want to update the URLs when this fork diverges further.

### CODE REVIEW

**Comment by KAMiKAZOW** [comment]
You should modify that file to point to this project. Upstream Aseprite is not responsible to provide support for this fork.

**Comment by Mailaender** [comment]
Updated this so it doesn't look like the fork is officially endorsed.

File: desktop/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Aseprite Desktop Integration Module
 # Copyright (C) 2016  Gabriel Rauter
 # Copyright (C) 2016  David Capello
+# Copyright (C) 2016  Matthias Mailänder
```
Comment (akien-mga) [inline desktop/CMakeLists.txt]: 3 copyright mentions for 15 functional lines? :D

Comment (Mailaender) [inline desktop/CMakeLists.txt]: At least I didn't have to sign a lengthy contract.

**Pull request merged**